tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
-            "es6"
+            "es6",
+            "DOM"
```

Comment:
This seems strange and I don't get why we would need `DOM` here (I mean I get that we need it here to make things work but I don't know _why don't they work without it_ and _is this the right fix_).  Is there documentation that prescribes this?  If not, can we check in with the team that this is what they expect?

Reply:
Oh never mind I realised you had already posted the link and oh dear yeah that's not good.

Reply:
😊 Yes, that is almost year and above when folks found the issue in “ms-rest-js” , so essentially deprecation help page should link the existing huge for folks who are fixing this deprecation to know what trick will actually fix their typing issues. 😎🙏